### DISCUSSION TITLE
how to fetch all the fields?

### DISCUSSION THREAD

**Discussion by rajamoulimallareddy** [prisma/prisma#16680] [Q&A]
how to fetch all the data where a particular document id is true

**Comment by nurul3101** [comment]
Hey @rajamoulimallareddy 👋 

Can you elaborate on what you mean by all data?
By default, all the scalar fields of the document are returned, if you also want to fetch the relation model's data, then you can use the [include](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#include) clause. 

Please feel free to share the schema file and your exact use case in case you have any confusion.

**Reply by pantharshit00** [reply]
👋 Hello @rajamoulimallareddy, did you perhaps have a chance to check @nurul3101's recommendation? Let us know if you still have any questions!
 
Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
